src/posthog-core.ts

```diff
@@ -145,6 +145,7 @@ export const defaultConfig = (): PostHogConfig => ({
     disable_persistence: false,
     disable_web_experiments: true, // disabled in beta.
     disable_surveys: false,
+    disable_survey_partial_response: false,
```

Comment:
```suggestion
    disable_survey_partial_response: true,
```

should be disabled by default right?